test/WebApp.Test/Api/Webfinger/WebfingerEndpointsTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AutoFixture.Xunit2;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Smilodon.WebApp.Api.Webfinger;
+
+namespace WebApp.Test.Api.Webfinger
+{
+    public class WebfingerEndpointsTests
+    {
+        [Theory, AutoData]
+        public async Task GetAllAccountReturnsResource(string resourceName)
+        {
+            var result = await WebfingerEndpoints.GetAllAccount(resourceName);
+            result.Should().BeOfType<Ok<string>>()
+                .Which.Value.Should().Be(resourceName);
```

Comment:
🤣 Glad to see my "hello world" works!

Reply:
It was either that or my usual "setting up a new unit test project and be sure they work" test of `true.Should().BeTrue();` 🤣